robottelo/ui/computeresource.py

```diff
@@ -361,6 +361,20 @@ def search_vm(self, resource_name, vm_name):
         strategy, value = self._search_locator()
         return self.wait_until_element((strategy, value % vm_name))
 
+    def power_on_status(self, resource_name, vm_name):
+        """Return the compute resource image power status
```

Comment:
`Return the compute resource VM power status`

It is not a image , but a actually vm running on RHEV